tools/f18/f18.cc

```diff
@@ -405,6 +416,10 @@ int main(int argc, char *const argv[]) {
       driver.dumpUnparse = true;
     } else if (arg == "-funparse-with-symbols") {
       driver.dumpUnparseWithSymbols = true;
+    } else if (arg == "-fdotty") {
+      driver.dumpGraph = true;
+    } else if (arg == "-fdebug-dump-linear-ir") {
+      driver.debugLinearFIR = true;
```

Comment:
The help message should be updated with these new options too.

Reply:
ok